snykTask/src/index.ts

```diff
@@ -199,28 +233,15 @@ async function runSnykTest(
   }
   const snykOutput: SnykOutput = { code: code, message: errorMsg };
 
-  // handle snyk code no-issues-found non-existent json by rerunning --json stdout to piped file
+  // handle bug in snyk CLI for code test, when --json-file-output is specified
```

Comment:
Nitpick: I know it is a nitpick, but the current behaviour is officially documented so it is more an inconsistency than a bug :)

Reply:
changed to "inconsistency"